Checkout3DSPackages/dummy.swift

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is there any online reference to why we are adding this dummy swift file and its purpose ?